tools/scripts/build-tests/build-al2-release-default.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+
+# This script is an example of how to build the SDK on Release build-type with minimize on
+# Directories created and files are prefixed with PREFIX_DIR argument
+# A clone of aws-sdk-cpp is expected to be in ${PREFIX_DIR}/aws-sdk-cpp
+# Platform: Amazon Linux 2
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: ${0} PREFIX_DIR"
+  exit 1
+fi
+PREFIX_DIR="$1"
+
+mkdir "${PREFIX_DIR}/al2-build"
+mkdir "${PREFIX_DIR}/al2-install"
+cd "${PREFIX_DIR}/al2-build"
+cmake -GNinja ../aws-sdk-cpp -DCMAKE_BUILD_TYPE=Release -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${PREFIX_DIR}/al2-install" -DBUILD_PERFORMANCE_TESTS=ON
```

Comment:
I dont think we should be adding a entire file for one arg change. cant we just use the [old script](https://github.com/aws/aws-sdk-cpp/blob/main/tools/scripts/build-tests/build-al2-debug-default.sh) but add a argument for build type, and have it default to debug if not provided?